omc3/model/constants.py

```diff
@@ -23,6 +23,7 @@
 
 GENERAL_MACROS = "general.macros.madx"
 LHC_MACROS = "lhc.macros.madx"
+LHC_MACROS_RUN3 = "lhc.macrosRun3.madx"
```

Comment:
Potentially `lhc.macros.run3.madx` for consistency

Reply:
Thanks! fixed!